rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2019-11-06
+nightly-2020-01-26
```

Comment:
Not opposed, but I don't entirely understand the implications of this.

Reply:
Just syncing this up to the version I am adding in other prs to rust-fil-proofs. No big implications atm, just making sure we keep up to date